feconf.py

```diff
@@ -114,10 +114,10 @@
 MAX_LEARNER_PLAYLIST_ACTIVITY_COUNT = 10
 
 # The minimum number of training samples required for training a classifier.
-MIN_TOTAL_TRAINING_EXAMPLES = 50
+MIN_TOTAL_TRAINING_EXAMPLES = 5
```

Comment:
If you're writing a backend test, then you can use self.swap as @prasanna08 says.

If this is just for convenience of testing locally, then change it when testing locally and change it back to the intended value in production before submitting to GitHub. The value here should be the correct one, or as near to it as possible. If we don't know the correct value then we should figure out the correct value.